readme/coding_style.md

```diff
@@ -0,0 +1,170 @@
+> Coding style is mostly enforced by a pre-commit hook that runs `eslint`. This hook is installed whenever running `yarn install` on any of the application directory. If for some reason the pre-commit hook didn't get installed, you can manually install it by running `yarn install` at the root of the repository.
```

Comment:
Please don't use blockquotes, even if it's quoted from elsewhere it's ok to add it without blockquote.